CONTRIBUTING.md

```diff
@@ -46,13 +46,36 @@ See the [contributor's testing guide](https://github.com/kudobuilder/kudo/blob/m
 
 ## Community, Discussion, and Support
 
-Learn how to engage with the Kubernetes community on the [community page](https://kudo.dev/community/).
+Learn how to engage with the KUDO community on the [community page](https://kudo.dev/community/).
 
 You can reach the maintainers of this project at:
 
 - [Slack](https://kubernetes.slack.com/messages/kudo/)
 - [Mailing List](https://groups.google.com/d/forum/kudobuilder)
 
+## Code culture
+
+This is a set of practices we try to live by when developing KUDO. These are just defaults (soft rules). Deviations from them are possible, only has to be justified.
+
+### General guidelines
+- Master is always releasable (green CI)
+- All feature/bug-fixing work should have an open issue with a description
```

Comment:
This could be understood as "all changes require an issue", which I think is pretty limiting, since barely anyone will bother fixing typos and doing other small improvements if it requires the hassle of filing an issue beforehand.

Here are some examples from recent history, of PRs which do not reference any issue.

https://github.com/kudobuilder/kudo/pull/1090
https://github.com/kudobuilder/kudo/pull/1075
https://github.com/kudobuilder/kudo/pull/1047
https://github.com/kudobuilder/kudo/pull/1069
https://github.com/kudobuilder/kudo/pull/1068
https://github.com/kudobuilder/kudo/pull/1039
https://github.com/kudobuilder/kudo/pull/1038
https://github.com/kudobuilder/kudo/pull/1037
https://github.com/kudobuilder/kudo/pull/1035

Can we somehow try to define in what cases this is acceptable?

Reply:
I kind of feel like we should aim for having issues in 99% cases (except for typos maybe? :D) - we might now be doing that right now. I guess we need to point it out in reviews if you feel like the issue is needed and refresh on this principle like that? Rather than making the rules with more explicit boundaries?

Reply:
There's definitely value in having a tracking issue for things that span multiple PRs (or has a KEP being worked on alongside), or if we want to have discussion before kicking off a PR (which should be almost all the time for something significant). Not sure how to write that down -- totally understand the desire to allow people to put in small PRs without having to go through the effort of _additionally_ opening an issue.

Reply:
How about this:
```suggestion
- All feature/bug-fixing work should have an open issue with a description, unless it's something very simple
```